### PR TITLE
Calibrationd: make recalibrating alert

### PR DESCRIPTION
**Pull request by haraschax**

### CODE REVIEW

File: selfdrive/controls/lib/events.py [hunk 1]
```diff
@@ -722,6 +722,12 @@ def joystick_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster,
     ET.SOFT_DISABLE: soft_disable_alert("Device remount detected: recalibrating"),
     ET.NO_ENTRY: NoEntryAlert("Calibration in Progress"),
   },
+  
+  EventName.calibrationRecalibrating: {
+    ET.PERMANENT: calibration_incomplete_alert,
+    ET.SOFT_DISABLE: soft_disable_alert("Device remount detected: recalibrating"),
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/events.py:728]: this doesn't match the capitalization style of the other alerts

File: selfdrive/controls/lib/events.py [hunk 2]
```diff
@@ -242,8 +242,9 @@ def below_steer_speed_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.S
 
 
 def calibration_incomplete_alert(CP: car.CarParams, CS: car.CarState, sm: messaging.SubMaster, metric: bool, soft_disable_time: int) -> Alert:
+  first_word = 'Recalibration' if sm['liveCalibration'].calStatus == 'recalibrating' else 'Calibration'
```
Comment (adeebshihadeh) [inline selfdrive/controls/lib/events.py:245]:
```suggestion
  first_word = 'Recalibration' if sm['liveCalibration'].calStatus == log.LiveCalibrationData.Status.recalibrating else 'Calibration'
```

File: selfdrive/ui/qt/offroad/settings.cc [hunk 3]
```diff
@@ -273,7 +273,7 @@ void DevicePanel::updateCalibDescription() {
       AlignedBuffer aligned_buf;
       capnp::FlatArrayMessageReader cmsg(aligned_buf.align(calib_bytes.data(), calib_bytes.size()));
       auto calib = cmsg.getRoot<cereal::Event>().getLiveCalibration();
-      if (calib.getCalStatus() != 0) {
+      if (calib.getCalStatus() != cereal::LiveCalibrationData::Status::UNCALIBRATED) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/settings.cc:276]: want this to show while recalibrating too?

Comment (HaraldSchafer) [reply selfdrive/ui/qt/offroad/settings.cc:276]: No that's fine, if recalibrating it already wrote new values.

**Pull request merged**